x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/condition_entry_input/index.tsx

```diff
@@ -213,7 +213,7 @@ export const ConditionEntryInput = memo<ConditionEntryInputProps>(
               color="danger"
               iconType="trash"
               onClick={handleRemoveClick}
-              isDisabled={isRemoveDisabled}
+              disabled={isRemoveDisabled}
```

Comment:
The EUI documentation still shows the prop `isDisabled` for [EuiButtonIcon](https://elastic.github.io/eui/#/navigation/button#icon-buttons). Can you confirm this change is correct?

Reply:
`disabled` is also valid for EuiButtonIcon, and this change is necessary until https://github.com/elastic/eui/pull/5022 is merged.
I'll add a comment if you'd like this change reverted in the next release.

Reply:
@thompsongl great! I just wanted to be sure this was not done by mistake. Thanks!

Reply:
Thanks for calling this out. It could have greater impact beyond than I originally thought. It's likely I'll make a patch release with elastic/eui#5022 and bump this PR to mitigate

Reply:
Reverted in b77ff9a